src/com/ar4android/AR4AndroidApplication.java

```diff
@@ -1,83 +1,157 @@
 	// A flag indicating if the scene has been already initialized.
 	private boolean mSceneInitialized = false;
 	// A flag indicating if a new Android camera image is available.
-	boolean	mNewCameraFrameAvailable = false;
+	boolean mNewCameraFrameAvailable = false;
 
+	private float mForegroundCamFOVY = 50; // for a Samsung Galaxy SII
+	
+	// for animation	
+	// The controller allows access to the animation sequences of the model
+	private AnimControl mAniControl;
+	// the channel is used to run one animation sequence at a time
+	private AnimChannel mAniChannel;
+	
+  
 	public static void main(String[] args) {
 		AR4AndroidApplication app = new AR4AndroidApplication();
 		app.start();
 	}
 
-	// The default method used to initialize your JME application.
 	@Override
 	public void simpleInitApp() {
-		// Do not display statistics or frames per second
+		// Do not display statistics
 		setDisplayStatView(false);
 		setDisplayFps(false);
-		// We use custom viewports - so the main viewport does not need to contain the rootNode
+		// we use our custom viewports - so the main viewport does not need the  rootNode
 		viewPort.detachScene(rootNode);
 		initVideoBackground(settings.getWidth(), settings.getHeight());
+		initForegroundScene();	
+		initBackgroundCamera();		
+		initForegroundCamera(mForegroundCamFOVY);
 	}
 
 	// This function creates the geometry, the viewport and the virtual camera
 	// needed for rendering the incoming Android camera frames in the scene
 	// graph
 	public void initVideoBackground(int screenWidth, int screenHeight) {
-		// Create a Quad shape. 
+		// Create a Quad shape.
 		Quad videoBGQuad = new Quad(1, 1, true);
 		// Create a Geometry with the Quad shape
-		mVideoBGGeom = new Geometry("quad", videoBGQuad);		 
-		float newWidth = 1.f * screenWidth / screenHeight;		
+		mVideoBGGeom = new Geometry("quad", videoBGQuad);
+		float newWidth = 1.f * screenWidth / screenHeight;
 		// Center the Geometry in the middle of the screen.
 		mVideoBGGeom.setLocalTranslation(-0.5f * newWidth, -0.5f, 0.f);//
-		// Scale (stretch) the width of the Geometry to cover the whole screen width.
+		// Scale (stretch) the width of the Geometry to cover the whole screen
+		// width.
 		mVideoBGGeom.setLocalScale(1.f * newWidth, 1.f, 1);
 		// Apply a unshaded material which we will use for texturing.
-		mvideoBGMat = new Material(assetManager, "Common/MatDefs/Misc/Unshaded.j3md");
+		mvideoBGMat = new Material(assetManager,
+				"Common/MatDefs/Misc/Unshaded.j3md");
 		mVideoBGGeom.setMaterial(mvideoBGMat);
-		// Create a new texture which will hold the Android camera preview frame pixels.
+		// Create a new texture which will hold the Android camera preview frame
+		// pixels.
 		mCameraTexture = new Texture2D();
+
 		
+		mSceneInitialized = true;
+	}
+	
+	public void initBackgroundCamera() {
 		// Create a custom virtual camera with orthographic projection
-		Camera videoBGCam = cam.clone();
+		Camera videoBGCam = cam.clone();		
```

Comment:
For the video background we used an orthographic camera to avoid perspective foreshortening of the video image